js/providers/Model.js

```diff
@@ -352,6 +354,13 @@ define(
 			*/
 			}
 
+			handleViewChange(view, routerParams = {}) {
+				if (routerParams !== false) {
```

Comment:
I don't understand the routerParams !== false here.  do you need to pass 'false' as the routerParams value to get that block to execute?  Instead should we just default routerParams to null and if we don't have a value, then don't set the routerParams observable?

Reply:
That's true, route params should be flushed between top-level page navigations unless you implicitly say the router not to do so (pass false as the second param).